qemu/tests/x86_cpu_flags.py

```diff
@@ -36,7 +38,17 @@ def run(test, params, env):
     error_context.context("Try to log into guest", logging.info)
     session = vm.wait_for_login()
     if params["os_type"] == "linux":
+        check_guest_cmd = params.get("check_guest_cmd", "")
```

Comment:
The default value "" seems to be meaningless.

Reply:
update and thanks.